pythran/analyses/constant_expressions.py

```diff
@@ -25,32 +25,32 @@ def add(self, node):
         return True
 
     def visit_BoolOp(self, node):
-        return all(map(self.visit, node.values)) and self.add(node)
+        return all([self.visit(x) for x in node.values]) and self.add(node)
```

Comment:
I can as all() stop evaluating generator as soon as it evaluate a value to false and I want to evaluate everything.